jbang-catalog.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
we shouldn't have this in main repo; can add it to `jbang-examples`
and we should create a `jbang-catalog`(?) for more "real" code.

Letting you just do `jbang asciidoctor@jbang` vs `jbang vertx@jbang/jbang-examples`

Reply:
> `jbang asciidoctor@jbang`

"jbang" or "jbangdev"? If "jbang" we'd have to hard-code this in the code (don't see a problem with that).

But...

> `jbang vertx@jbang/jbang-examples`

This might be harder to justify. You suggested being able to use GitHUb (and GitLab/BItBucket) repo names here, which is what this looks like. But then it should be `jbang vertx@jbangdev/jbang-examples`, right? If you really want `jbang vertx@jbang/jbang-examples` instead we'd have to come up with some logical explanation for that, which I can't find right now :-)

Reply:
i blame it on my current flu :) it should be asciidoctor@jbangdev , i.e. `<catalogentry>@<githuborg>[/reponame]` where if reponame is missing is `jbang-catalog`

Reply:
Ok, perfect